bin/prepare-release.sh

```diff
@@ -160,7 +170,7 @@ tar --exclude .git \
     --exclude Dockerfile-Production-Release \
     --exclude docker-compose.yml \
     --exclude docker-compose.dump.yml \
-    -cvzf "${OUTPUT_DIR}$LANSUITE_FILENAME.tar.gz" .
+    -czf "${OUTPUT_DIR}$LANSUITE_FILENAME.tar.gz" .
```

Comment:
`zip`may be a better format here, although I would see leave as-is and see if we get any requests / questions in that regard